packages/js/e2e-core-tests/specs/data/elements.js

```diff
@@ -0,0 +1,85 @@
+/**
+ * WP top-level menu items and their associated sub-menus
+ */
+
+export const MENUS = [
+	[
+		'WooCommerce',
+		'#adminmenu > li:nth-child(8) > a',
+		[
+			[
+				'Home',
+				'#toplevel_page_woocommerce > ul > li:nth-child(2)',
+				'Home',
+			],
+			[
+				'Orders',
+				'#toplevel_page_woocommerce > ul > li:nth-child(3)',
+				'Orders',
+			],
+			[
+				'Reports',
+				'#toplevel_page_woocommerce > ul > li:nth-child(6)',
+				'Orders',
+			],
+			[
+				'Settings',
+				'#toplevel_page_woocommerce > ul > li:nth-child(7)',
+				'General',
+			],
+			[
+				'Status',
+				'#toplevel_page_woocommerce > ul > li:nth-child(8)',
+				'System status',
+			],
+			// [ 'Extensions', '#toplevel_page_woocommerce > ul > li:nth-child(9)', 'Extensions' ],
```

Comment:
@TODO

Uncomment this line when the issue outlined [here](https://github.com/woocommerce/woocommerce/issues/31304) is fixed.